client/state/stored-cards/reducer.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	STORED_CARDS_FETCH,
+	STORED_CARDS_FETCH_COMPLETED,
+	STORED_CARDS_FETCH_FAILED,
+	STORED_CARDS_DELETE,
+	STORED_CARDS_DELETE_COMPLETED,
+	STORED_CARDS_DELETE_FAILED
+} from 'state/action-types';
+
+import { createStoredCardsArray } from './assembler.js';
+
+/**
+ * List all known stored cards of the current user at /me/stored-cards.
```

Comment:
Why is that? Maybe the `at /me/stored-cards` is confusing here. I added it because I found the info valuable (since you rarely know from looking at the reducer from which endpoint the data is pulled from)